edx_event_bus_kafka/internal/producer.py

```diff
@@ -36,6 +38,17 @@
 EVENT_TYPE_HEADER_KEY = "ce_type"
 
 
+def record_error(error, context):
+    """
+    Record an error in producing an event to both the monitoring system and the regular logs
+    """
+    try:
+        raise Exception(error)
+    except BaseException:
+        record_exception()
+        logger.exception(f"Error publishing event to event bus. {error=!s} {context!r}")
```

Comment:
We use the term "produce" everywhere, except in the error message. This is the only place we use "publish"? Can you use some form of "producing" instead?

Reply:
[fixed]